F1WM/Controllers/NewsController.cs

```diff
@@ -13,18 +13,29 @@ namespace F1WM.Controllers
 	[Route("api/[controller]")]
 	public class NewsController : ControllerBase
 	{
-		private const int defaultLatestNewsCount = 20;
+		private const int defaultPage = 1;
+		private const int defaultCountPerPage = 20;
 
 		private readonly INewsService service;
 		private readonly ILoggingService logger;
 
 		[HttpGet]
-		public async Task<IEnumerable<NewsSummary>> GetMany(
-			[FromQuery(Name = "firstId")] int? firstId = null, [FromQuery(Name = "count")] int count = defaultLatestNewsCount)
+		public async Task<NewsSummaryPaged> GetMany(
```

Comment:
After adding several new methods in the controller, it would make more sense to rename this one to GetManyNews for example.